.000my_spiders/jimmy_fairly.py

```
@@ -0,0 +1,54 @@
+        return oh
+
+    def post_process_item(self, item: Feature, response: TextResponse, location: dict) -> Iterable[Feature]:
+        item["branch"] = item.pop("name", "").removeprefix("Jimmy Fairly - ")
+        if location.get("url") is not None:
+            item["website"] = "https://www.jimmyfairly.com" + location["url"]
+
+        if location.get("opening_hours") is not None:
+            try:
+                item["opening_hours"] = self.parse_opening_hours(location["opening_hours"])
+            except Exception:
+                pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)